smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -12664,7 +12665,7 @@ StackInterpreter >> pushNewArrayBytecode [
 
 { #category : #'stack bytecodes' }
 StackInterpreter >> pushReceiverBytecode [
-	<needsFrameNever: 1>
+	"<needsFrameNever: 1>"
```

Comment:
???

Reply:
In the simple stack jit pushing the receiver sometimes needs a frame, sometimes not. So the annotation is not right for it.

I want to discuss this with you :)